tests/losses/test_masked_loss.py

```
@@ -73,14 +73,14 @@ def test_ill_opts(self):
             masked = MaskedLoss(loss=dice_loss)
             masked(input=torch.zeros((3, 3, 2, 2)), target=torch.zeros((3, 2, 2, 2)), mask=torch.zeros((3, 3, 2, 2)))
 
-    def test_script(self):
+    def test_export(self):
         input_param, expected_val = TEST_CASES[0]
```

Ruff (0.15.6)

[warning] Unpacked variable `expected_val` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)